cmd/errtrace/main.go

```diff
@@ -920,9 +919,33 @@ func (t *walker) deferStmt(n *ast.DeferStmt) {
 				continue // not an error assignment
 			}
 
-			// Assigning to a named error return value.
-			// Wrap the rhs in-place.
-			t.wrapExpr(1, assign.Rhs[i])
+			// Assignment to an error return value.
+			// This will take one of the following forms:
+			//
+			//  (1) x, y, err := f1(), f2(), f3()
+			//  (2) x, y, err := f() // returns multiple values
+			//  (3) x, err, z := f() // returns multiple values
```

Comment:
comment nit: as discussed offline, these would probably be `=` rather than `:=` since that would create a new variable, so would be skipped by the above `t.errorObjs` check